### PR TITLE
events/m3: Add skeleton for presentation slides

### PR DESCRIPTION
**Pull request by RazvanN7**
cc @teodutu @razvand

### CODE REVIEW

File: website/docs/events/m3-slides/.slides/next-steps.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+
+## Next Steps
+
+---
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/next-steps.md]:
```suggestion
```

File: website/docs/events/m3-slides/.slides/os-materials.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+### Quizzes
+
+---
+
+### Demos
+
+---
+
+### Homework
```
Comment (teodutu) [inline website/docs/events/m3-slides/.slides/os-materials.md:23]:
```suggestion
### Homework Assignments
```

**Review by teodutu — CHANGES_REQUESTED** [review]
The slides build and render fine. Do you want us to add content to them?

**Comment by RazvanN7** [comment]
@teodutu @razvand and myself will fill those.

File: website/docs/events/m3-slides/.slides/os-materials.md (hunk 2 above)
Comment (RazvanN7) [reply website/docs/events/m3-slides/.slides/os-materials.md:23]: These are just for orientation. The slide could even disappear in the final version.

**Comment by RazvanN7** [comment]
@teodutu please merge this.

**Review by teodutu — APPROVED** [review]

**Comment by teodutu** [comment]
I can't merge it until you squash your commits as squashing during merge is disabled.

**Comment by RazvanN7** [comment]
> I can't merge it until you squash your commits as squashing during merge is disabled.

That's just dumb on our side.

**Comment by teodutu** [comment]
We should probably stick to naming our commits like `events/m3: Add skeleton slides`. 3rd `git push -f` is the charm

**Comment by RazvanN7** [comment]
I've enabled squash during merge

**Comment by teodutu** [comment]
> I've enabled squash during merge

Then nvm, I'll reword the commit message when I merge it.

**Pull request merged**